pkg/system/system.go

```diff
@@ -273,6 +274,20 @@ func RunDelete(cmd *cobra.Command, args []string) {
 
 	log.Infof("Notice: Deletion of External secrets should be preformed manually")
 
+	scopeNamespaced := admissionv1.ScopeType("Namespaced")
+	rule := &admissionv1.RuleWithOperations{
```

Comment:
can we reuse this code chunk? it appears 3 times

Reply:
Done ✔️